airlock/templates/file_browser/file_content/csv.html

```diff
@@ -21,6 +27,7 @@
         {% #datatable column_filter searchable sortable %}
           <thead>
             <tr>
+              <th class="datatable-row-number" data-searchable="false" data-sortable="false"></th>
```

Comment:
Do we definitely not want this column to be sortable?  I've found it annoying elsewhere when I've not been able to get a table back into its initial state after sorting it.